modules/packages/Sort.chpl

```diff
@@ -1233,52 +1233,62 @@ module QuickSort {
     import Sort.InsertionSort;
 
     // grab obvious indices
-    const lo = start,
-          hi = end,
-          mid = lo + (hi-lo+1)/2;
-    var piv = mid;
-
-    if hi - lo < 0 { // minlen {
-      // base case -- use insertion sort
-      InsertionSort.insertionSortMoveElts(Data, comparator=comparator, lo, hi);
-      return;
-    } else if hi <= lo {
-      // nothing to sort
-      return;
-    }
+    var lo = start,
+          hi = end;
 
-    // find pivot using median-of-3 method for small arrays
-    // and a "ninther" for bigger arrays. Places the pivot in
-    // Data[lo].
-    if hi - lo < 100 {
-      piv = order3(Data, lo, mid, hi, comparator);
-    } else {
-      // assumes array size > 9 at the very least
 
-      // median of each group of 3
-      const medLo  = order3(Data, lo,    lo+1, lo+2,  comparator);
-      const medMid = order3(Data, mid-1, mid,  mid+1, comparator);
-      const medHi  = order3(Data, hi-2,  hi-1, hi,    comparator);
-      // median of the medians
-      piv = order3(Data, medLo, medMid, medHi, comparator);
-    }
+    while(lo <= hi){
```

Comment:
If there's just one element, it's in order, right? So this can be `while lo < hi {`

Reply:
Yeah missed this